ddsc/ddsclient.py

```diff
@@ -222,3 +227,47 @@ def get_new_project_name(self, project_name):
         timestamp_str = datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M')
         return "{} {}".format(project_name, timestamp_str)
 
+
+class ListCommand(object):
+    """
+    Print out a list of project names one line at a time.
+    Names are escaped so the output can be used with the delete command.
+    """
+    def __init__(self, config):
+        """
+        Pass in the config for which data service and user to list data for.
+        :param config: Config global configuration for use with this command.
+        """
+        self.remote_store = RemoteStore(config)
+
+    def run(self, args):
+        """
+        Lists project names.
+        :param args Namespace arguments parsed from the command line
+        """
+        names = self.remote_store.get_project_names()
+        if names:
+            for name in names:
+                print(pipes.quote(name))
```

Comment:
I wanted list and delete to work together.
So that any name printed by list would be directly pluggable into delete.
I assumed they would copy a project from the output of `ddsclient list` and paste it in `ddsclient delete -p <HERE>`.